service/controller/resource/namespaceauth/create.go

```diff
@@ -0,0 +1,66 @@
+package namespaceauth
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/giantswarm/rbac-operator/service/controller/key"
+)
+
+func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
+	namespace, err := key.ToNamespace(obj)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	resources, err := r.k8sClient.Discovery().ServerPreferredResources()
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	viewAllRole, err := newViewAllRole(resources)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	r.logger.LogCtx(ctx, "level", "debug", "message", "creating view role")
```

Comment:
Can we please add namespace and name to those logs?

```suggestion
	r.logger.LogCtx(ctx, "level", "debug", "message", "creating view role %#q in namespace %#q", viewAllRole.Name, namespace.Name)
```